### ISSUE TITLE
Solicitud de tamaño en almacenamiento de objetos

### ISSUE THREAD

**Issue by acastrovargas** [entrywan/cli#1]
Es una opción interesante agregar a CLI " entrywan size bucket_name "

**Comment by aarongroom** [comment]
Good idea.  That can be implemented once the authentication regime for object storage and the global API are unified.
